extensions/ql-vscode/src/authentication.ts

```diff
@@ -54,11 +48,11 @@ export class Credentials {
       return this.octokit;
     }
 
-    await this.initializeOctokit(true);
+    this.octokit = await this.createOctokit(true);
     // octokit shouldn't be undefined, since we've set "createIfNone: true".
     // The following block is mainly here to prevent a compiler error.
```

Comment:
RIght...I understand now. I think it's best to keep the if-block. We _could_ use a type assertion to force the compiler to ignore the possibly undefined return value: `this.octokit = await this.createOctokit(true)!;` But, I think keeping the if-block and the comment is safer.